translator.py

```diff
@@ -0,0 +1,129 @@
+import os
+word = str("")
+final_word = str(input())
+while final_word != word:
+    Trigger = float(input("Enter your Trigger: "))
```

Comment:
Convention for variables is "[snake case](https://en.wikipedia.org/wiki/Snake_case)", some of your variables are snake case, and some others are not. Stick with the convention.